### PR TITLE
chore(deps): update

### PR DESCRIPTION
**Pull request by dargmuesli**
Waiting for:
- https://github.com/nuxt/scripts/issues/133

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,33 +53,33 @@
...
+    "nuxt": "3.12.3",
+    "playwright": "1.45.1",
+    "typescript": "5.5.3",
     "vitest": "1.6.0",
-    "vue-tsc": "2.0.24"
+    "vue-tsc": "2.0.26"
   },
   "peerDependencies": {
-    "@nuxt/scripts": ">=0.5.0 || <0.6.0"
+    "@nuxt/scripts": ">=0.6.0 || <0.7.0"
```
Comment (danielroe) [inline package.json:82]: Won't this match every possible version?

Comment (dargmuesli) [reply package.json:82]: Whoops, you're correct. I think `^0.6.0` is what I mean which should be an alias for `>=0.6.0 <0.7.0`. It should stay below 0.7.0 as that would likely contain breaking changes. Do you agree?

Comment (danielroe) [reply package.json:82]: Yes 👍

Comment (dargmuesli) [reply package.json]: changed :raised_hands:

**Pull request merged**